modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_entries.jsp

```diff
@@ -141,7 +141,7 @@ DLViewEntriesDisplayContext dlViewEntriesDisplayContext = new DLViewEntriesDispl
 											<div class="card-row">
 												<div class="autofit-col autofit-col-expand">
 													<aui:a cssClass="card-title text-truncate" href="<%= dlViewEntriesDisplayContext.getViewFileEntryURL(fileEntry) %>" title="<%= HtmlUtil.escapeAttribute(latestFileVersion.getTitle()) %>">
-														<%= HtmlUtil.escape(latestFileVersion.getTitle()) %>
+														<%= HtmlUtil.unescape(latestFileVersion.getTitle()) %>
```

Comment:
Why unescape this? Shouldn't you simply remove the call to `escape`? Otherwise, you open this to an XSS attack.

Reply:
I've follow the same pattern as other cases.

Reply:
The first case will be fed into Js code (that I hope will handle the escaping); but in this second case, you're taking already escaped content, unescaping it and embeeding as is in HTML. I think it should not be unescaped.

Reply:
So we have to change all this cases?

Reply:
No, for what I've seen, this is the only case. The file version is escaped in the display context, so there's no need to do anything extra with it. Just don't double escape it.

Reply:
updated